tools/run_tests/xds_k8s_test_driver/framework/infrastructure/gcp/compute.py

```diff
@@ -190,15 +190,15 @@ def backend_service_patch_backends(
         backends,
         max_rate_per_endpoint: Optional[int] = None,
     ):
-        if max_rate_per_endpoint is None:
-            max_rate_per_endpoint = 5
+        backend_attrs = {
+            "balancingMode": "RATE",
+        }
+        if max_rate_per_endpoint is not None:
```

Comment:
[Minor] We can just use `if max_rate_per_endpoint:`

Reply:
Not this time - that would not work with integer `0`